frontend/src/app.js

```diff
@@ -0,0 +1,48 @@
+
+import 'babel-polyfill/browser';
+import es6Promise from 'es6-promise';
+import 'isomorphic-fetch';
+import 'l20n/dist/compat/web/l20n';
```

Comment:
Oh, I thought this was fixed during the rebase, but the fix must have gotten lost. Thanks for finding it!